### PR TITLE
Handling eslint warnings - fixes #77

### LINKED ISSUE #77: How to handle eslint rules id-length and prefer-rest-params
These two rules are currently checked with warnings. We have to discuss how we handle these.

### PR DESCRIPTION
**Pull request by inpercima**
* define some more intuitive names of variables
* disable one line from check (prefer-rest-params)
* fix no-nested-ternary

### CODE REVIEW

File: src/main/resources/public/js/date.time.utils.js [hunk 1]
```diff
@@ -28,7 +28,8 @@
     }
 
     function getTimePart(part) {
-      return part < 1 ? '00' : (part < 10 ? `0${part}` : part);
+      const timePart = part < 1 ? '00' : part;
+      return part < 10 ? `0${part}` : timePart;
```
Comment (sepe81) [inline src/main/resources/public/js/date.time.utils.js]: IMO this change makes the expression overcomplicated -- the original form is better readable. If `part < 1` than we don't need to evaluate `part < 10` at all...

File: src/main/resources/public/js/log.config.js [hunk 2]
```diff
@@ -23,6 +23,8 @@
 
       function enhanceLogging(logger, className, logText) {
         return function() {
+          // the next line cannot simply refactored so it will be disabled from check
```
Comment (sepe81) [inline src/main/resources/public/js/log.config.js]: cannot be simply refactored

File: src/main/resources/public/js/date.time.utils.js (hunk 1 above)
Comment (inpercima) [inline src/main/resources/public/js/date.time.utils.js]: I will change it back and prevent this line from eslint checks.

**Comment by inpercima** [comment]
@sepe81 You can check this again.

**Pull request merged**